### ISSUE TITLE
About suitable questions

### ISSUE THREAD

**Issue by hazardout** [luka-group/vlm-knowledge-conflict#3]
Hi author!
I‘m pretty interested in your work! But I want to confirm whether your method is not applicable to VQA questions without factual answers.For example , will the method improve the question like "Is there an apple in the image?"
Because in my view, your method based on the knowledge of language model and visual model. So whether the method can't further improve the performance of the VQA problem when using the internal knowledge of the language model alone is not enough to answer some questions(like the query above)?
Thanks again for such an excellent job！！

**Comment by DarthZhu** [comment]
Hi @hazardout,

Thanks for your interest in our work!

Although we did not specifically experiment with VQA questions, I believe our approach can adapt to questions that heavily rely on visual context. For example, consider your question, "Is there an apple in the image?" In this case, the confidence of the textual modality is expected to be relatively low, while the confidence of the visual modality would be higher. By applying dynamic contrastive decoding, the model is likely to focus more on the visual context, as the textual logits are subtracted from the visual logits.

I think you can definitely try it on VQA questions and see how it performs. Feel free to reach out if you have further questions or would like to discuss this in more detail.

**Comment by hazardout** [comment]
Got it!  I will try it. Thanks for the reply!
